compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -301,6 +302,7 @@ int entry(int argc, char **argv)
   option_str_to_enum["fuse_preactivation_batchnorm"] = Algorithms::FusePreActivationBatchNorm;
   option_str_to_enum["fuse_prelu"] = Algorithms::FusePRelu;
   option_str_to_enum["fuse_gelu"] = Algorithms::FuseGelu;
+  option_str_to_enum["fuse_rmsnorm"] = Algorithms::FuseRmsNorm;
```

Comment:
plz add setting flag from luci/pass if this option is enabled.

Reply:
> plz add setting flag from luci/pass if this option is enabled.

Where's setting flag in luci pass? I couldn't understand it.

Reply:
Ah, there was refactoring and has changed ... -_-;